docassemble/ALWeaver/interview_generator.py

```diff
@@ -612,7 +624,7 @@ def review_yaml(self):
     content += '    button: |\n'
         
     if self.var_type == 'list': 
-      content += indent_by(bold(self.var_name), 6) + '\n'
+      content += indent_by(bold(self.var_name.capitalize().replace('_', '')), 6) + '\n'
```

Comment:
Is this meant to be read by a user? If so, was a space intended as the replacement?
```suggestion
      content += indent_by(bold(self.var_name.capitalize().replace('_', ' ')), 6) + '\n'
```

Reply:
Alternatively, would it be useful to use the already established var name guess?
```suggestion
      content += indent_by(bold(self.variable_name_guess.capitalize()), 6) + '\n'
```

Reply:
Yes, good call. I had caught some of those but it looks like I missed these